### ISSUE TITLE
create event at mouse coords on cell simple click

### ISSUE THREAD

**Issue by nature-li** [antoniandre/vue-cal-v4#94]
Hi @antoniandre ,
I want to create an event when I click an no-event cell but not hold a cell. could you add a cell-click callback to the vue-cal, like @cell-click=onCellClick? the callback paramter include the begining time of a cell but not the time specified by the cursor position.  

I saw the document at # Edit, delete & create events. But I don't want to make a cell editable and I must hold the cell for a while.

Or  could I do what I want using the current version?

**Comment by antoniandre** [comment]
Hi @nature-li,

Currently there is no possibility to get the cursor position in cell-click event. I will add it in a next release.

The cell-click event is called `day-click`. Refer to this: https://antoniandre.github.io/vue-cal/#ex--emitted-events

This emitted event will be renamed into `cell-click` very soon as I introduce the ability to get events on cell click in `years` and `year` views as well. Keep an eye on release notes! 
https://antoniandre.github.io/vue-cal/#release-notes

**Comment by CarlosOnline** [comment]
Vote for same feature request.  This feature would be very useful to me.

**Comment by CarlosOnline** [comment]
Was able to change the hold time out via hack below.

```
$refs.cal._data.domEvents.clickHoldACell.timeout = 30
```

In source code:
```js
clickHoldACell: {
    cellId: null,
    split: null,
    timeout: 1200, // Hold for 1.2s to create an event.
    timeoutId: null
}
```

**Comment by matheusmoura10** [comment]
Excellent, what is the forecast of this new release?

**Comment by antoniandre** [comment]
Hi @matheusmoura10, thanks for your enthusiasm.

There are a couple of things I need to finish before starting the drag & drop feature:

- At the moment I am refactoring the way multiple-day events work (43e35ad), and this is very important for the future recurring events and the overall good functioning of all "segmented" events. This will also fix #78 and change the way the overlaps work internally.
This is a big job and It needs to be done before moving to a more complex codebase.
- Externalizing locales (to reduce the library size) is already done but I need to activate it and it will slightly change the way the lib is used, so I will release a v2.0.0 at this point along with `v-slots` new syntax update from Vue which will require Vue@2.6.0+.

Along the way I will see if I can include an option to create events on simple click.

**Comment by antoniandre** [comment]
Hi @nature-li, @CarlosOnline, @matheusmoura10,
The V2 is out and this feature too!
Check this example: https://antoniandre.github.io/vue-cal/#ex--more-advanced-event-creation

Also don't forget to read the release notes!

Hope it helps.
